### ISSUE TITLE
Wrong days in calendar pane when timezone different than UTC

### ISSUE THREAD

**Issue by nass600** [nathanreyes/v-calendar#182]
Hello there,

I am observing the days are wrongly laid out in the calendar pane depending on the browser timezone.
Might be related to [https://github.com/nathanreyes/v-calendar/issues/50](this issue) but since it seems version `0.8.0` fixed that problem and I am having this troubles might be worth to comment.

To reproduce it you can take the example in `https://vcalendar.netlify.com/datepicker` and chrome version `68.0.3440.106`

**Test 1:**
 Without altering the timezone, the calendar renders the days correctly

+ Timezone: Europe/Dublin (my current timezone)
+ Screenshot taken on: 2018-08-14 at 13:35:58

<img width="738" alt="screen shot 2018-08-14 at 13 35 58" src="https://user-images.githubusercontent.com/759551/44092674-8142ada0-9f64-11e8-968a-a1be42667d17.png">

**Test 2:**
Installing the chrome extension to alter the timezone: https://chrome.google.com/webstore/detail/change-timezone-time-shif/nbofeaabhknfdcpoddmfckpokmncimpj?hl=en
The calendar days are shifted 1 day in the past no matter which timezone you set but for the ones that are equal to UTC:

<img width="259" alt="screen shot 2018-08-14 at 13 55 23" src="https://user-images.githubusercontent.com/759551/44092818-e572eff6-9f64-11e8-8cce-32f1675c4174.png">

This is the result:
+ Timezone: Pacific/Niue (altered by the extension)
+ Screenshot taken on: 2018-08-14 at 13.36.34

<img width="730" alt="screen shot 2018-08-14 at 13 36 34" src="https://user-images.githubusercontent.com/759551/44092680-85c6313a-9f64-11e8-847c-a6708ee2a3c4.png">

As you see the dates are shifted one day in the past.

**Comment by nass600** [comment]
It should be fixed in #183

**Comment by nass600** [comment]
This issue and the PR are invalid.
I found out the issue were coming from using `setupCalendar` in the `mounted` function. Depending on the device performance the `firstDayOfWeek` was shifting the days as the component was built and mounted already:

    mounted () {
        setupCalendar({
            firstDayOfWeek: 2,
            locale: this.locale
        });
    }

To fix it I moved the configuration within the `created` function and the days in the calendar are now properly laid out in any device and browser.
